src/lib/migration.ts

```diff
@@ -256,3 +264,87 @@ export async function normaliseLegacyDatabase(
 		legacyAppsPresent: false,
 	});
 }
+
+export async function loadBackupFromMigration(
+	deviceState: DeviceState,
+	targetState: TargetState,
+	retryDelay: number,
+): Promise<void> {
+	try {
+		const exists = await pathExistsOnHost(
+			path.join('mnt/data', constants.migrationBackupFile),
+		);
+		if (!exists) {
+			return;
+		}
+		log.info('Migration backup detected');
+
+		await deviceState.setTarget(targetState);
+
+		const appId = parseInt(_.keys(targetState.local?.apps)[0], 10);
```

Comment:
Can you put a multi-app warning here? I add them to the code for places that definitely won't support multiple applications

Reply:
ack